### PR TITLE
Refactor company to remarks

### LINKED ISSUE #201: Change companies to remarks

### PR DESCRIPTION
**Pull request by notle1706**

### CODE REVIEW

**Review by ElginL — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/remark/Remark.java [hunk 1]
```diff
@@ -13,32 +13,26 @@
  * Represents a Point of contact in the client.
```
Comment (Jo3LW) [inline src/main/java/seedu/address/model/remark/Remark.java]: Great job refactoring! I think you might have missed out on this class description. Maybe can remove that it represents a point of contact. All the best!

**Review by Jo3LW — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/remark/Remark.java (hunk 1 above)
Comment (notle1706) [reply src/main/java/seedu/address/model/remark/Remark.java]: Thank you for spotting this mistake! I'll make the changes.

**Pull request merged**